micro-rdk/src/common/camera/fake_camera.rs

```diff
@@ -94,15 +94,13 @@ mod tests {
             app_client::encode_request,
             config::DynamicComponentConfig,
             conn::server::{AsyncableTcpListener, Http2Connector},
+            exec::CPUBoundExecutor,
```

Comment:
I think I somewhat dislike this naming. It doesn't seem to me like any of this code actually depends on the CPU-bound-ness of the executor. At least I hope it doesn't. It should all work just as well if we had a executor that could move from core to core. We just don't happen to configure for that right now. Is there a reason that it can't just be `common::exec::Executor`?